backend/src/main/kotlin/com/tngtech/apicenter/backend/connector/rest/dto/SpecificationFileDto.kt

```diff
@@ -3,8 +3,15 @@ package com.tngtech.apicenter.backend.connector.rest.dto
 import com.fasterxml.jackson.annotation.JsonCreator
 import java.util.UUID
 
+data class SpecificationMetaData constructor(
```

Comment:
Are these really things that should be coupled to a `SpecificationFileDto`? Can the title switch between files? If so, would it make sense to have these things as fields in the `SpecificationFileDto`?